crates/aptos/src/lib.rs

```diff
@@ -1,11 +1,10 @@
 // Copyright © Aptos Foundation
 // SPDX-License-Identifier: Apache-2.0
 
-#![forbid(unsafe_code)]
```

Comment:
Since the FFI functions are unsafe. I have to remove this from the root lib.

May I move this to other sub-mods?

Reply:
Yeah, we probably want to apply this to the other sub mods.  Either that or if you can find a way to allow it only on the `ffi` module

Reply:
Ok, I think I found a work around. 

Adding `#![allow(unsafe_code)]` to ffi
Adding `#![deny(unsafe_code)]` to lib